CMake/SlicerCheckCMakeHTTPS.cmake

```diff
@@ -22,7 +22,8 @@ function(slicer_check_cmake_https)
     list(GET status 0 error_code)
     file(REMOVE ${dest})
     if(error_code)
-      message(FATAL_ERROR "error: This CMake does not support the HTTPS protocol. Ensure that CMake is compiled with CMAKE_USE_OPENSSL enabled.")
+      list(GET status 1 error_msg)
+      message(FATAL_ERROR "error: ${error_msg}")
```

Comment:
It could be still useful to tell users that the error is due to wrong CMake configuration and they may try to download a different version or build on their own. Or, if you don't want to complicate the error message then at least we could write some more information in comments here (developers can find the comments by looking up the filename&linenumber reported by CMake).